apps/wpcom-block-editor/src/common/tracking/track-record-event.js

```diff
@@ -1,20 +1,30 @@
 /**
  * External dependencies
  */
-import { registerPlugin } from '@wordpress/plugins';
 import { isObjectLike, isUndefined, omit } from 'lodash';
 import debug from 'debug';
+
 const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
 
-// In case Tracks hasn't loaded
+// In case Tracks hasn't loaded.
 if ( typeof window !== 'undefined' ) {
 	window._tkq = window._tkq || [];
 }
 
 // Adapted from the analytics lib :(
 // Because this is happening outside of the Calypso app we can't reuse the same lib
 // This means we don't have any extra props like user
-const tracksRecordEvent = function( eventName, eventProperties ) {
+
+export default ( eventName, eventProperties ) => {
+	if ( undefined === window || undefined === window._currentSiteId ) {
```

Comment:
We should test `window._currentSiteType` here, too. See CJS75TX3R/p1563482008221900-slack-ajax.